packages/task/src/browser/quick-open-task.ts

```diff
@@ -252,7 +252,7 @@ export class TaskRunQuickOpenItem extends QuickOpenGroupItem {
         }
         if (ContributedTaskConfiguration.is(this.task)) {
             if (this.task._scope) {
-                return new URI(this.task._scope).path.toString();
+                return new URI(this.task._scope).displayName;
```

Comment:
Do you think it'd be overkill to use the `labelProvider` in the following case instead?